README.md

```diff
@@ -52,31 +48,29 @@ To update these docs, you'll need to:
 4. Add back the YAML front-matter to each file that was changed
 5. Commit the changes and create a PR to update the website.
 
-
 ### How to Write a Blog Post
 
 Blog posts are created via pull requests. The following steps are used to add them:
 
-1) Add a new file to the `content/en/blog/` directory whose name is the published date and the title. The files must be markdown formatted. See the existing titles for examples of the format
-2) Add the header meta-data to the file using this format (note the permalink structure). Recommended but optional fields are `authorname` which should be name(s); these are displayed verbatim. `authorlink` is the link used by `authorname`.
+1. Add a new file to the `content/en/blog/` directory whose name is the published date and the title. The files must be markdown formatted. See the existing titles for examples of the format
+2. Add the header meta-data to the file using this format (note the permalink structure). Recommended but optional fields are `authorname` which should be name(s); these are displayed verbatim. `authorlink` is the link used by `authorname`.
 
-```yaml
----
-title: "A Fancy Title"
-slug: "fancy-title"
-authorname: "Captain Awesome"
-authorlink: "https://example.com"
-date: "yyyy-mm-dd"
----
-```
+    ```yaml
+        ---
+        title: "A Fancy Title"
+        slug: "fancy-title"
+        authorname: "Captain Awesome"
+        authorlink: "https://example.com"
+        date: "yyyy-mm-dd"
+        ---
+    ```
```

Comment:
Why is this section indented?

Reply:
Good catch. The section is indented to make it inline with point 2 in the ordered list. Otherwise markdownlint spews out an error saying point 3 should start with 1. In other words, they're treated as separate lists.

However it looks like the code block is now indented by 4 spaces. That was unintentional.